### ISSUE TITLE
Allow parantheses around logical expressions in R Programming, section 8

### ISSUE THREAD

**Issue by posto** [swirldev/swirl_courses#101]
`!(5==7)` is not recognized as a correct answer while `!5==7` is. Please allow for parantheses as well.

```
  |==========================                                                                     |  27%
| In order to negate boolean expressions you can use the NOT operator. An exclamation point `!` will
| cause !TRUE (say: not true) to evaluate to FALSE and !FALSE (say: not false) to evaluate to TRUE. Try
| using the NOT operator and the equals operator to find the opposite of whether 5 is equal to 7.

> !(5==7)
[1] TRUE

| That's not exactly what I'm looking for. Try again. Or, type info() for more options.

| This expression may be a little tricky, so think about negating the expression 5 == 7 (all you need is
| an exclamation point in front).

> !5==7
[1] TRUE

| That's the answer I was looking for.
```

**Comment by ncarchedi** [comment]
Yea, I agree we should allow both. Easy to do with the any_of_exprs() answer test.

**Comment by seankross** [comment]
Fixed in 5cca7876b45fa99f62dd8176caf57873faa1b493. Closing this issue.
